Sources/App/Resources/en.lproj/Localizable.strings

```diff
@@ -385,7 +389,8 @@ Home Assistant is free and open source home automation software with a focus on
 "settings_details.actions.actions_synced.footer" = "Actions defined in .yaml are not editable on device.";
 "settings_details.actions.actions_synced.footer_no_actions" = "Actions may be also defined in the .yaml configuration.";
 "settings_details.actions.actions_synced.header" = "Synced Actions";
-"settings_details.actions.footer" = "Actions are used in the Apple Watch app, App Icon Actions and the Today widget";
+"settings_details.actions.footer" = "Actions are used in the Apple Watch app, App Icon Actions, the Today widget and CarPlay.";
```

Comment:
you'll need to manually reset this in Lokalize

Reply:
Good reminder!